source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,59 @@
+.. _pymongo-get-started-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install Dependencies
+
+      Ensure you have Python3 version 3.7 or later,
+      pip, and dnspython installed in your development environment.
+
+      For information on how to install pip, see the
+      `pip Installation guide <https://pip.pypa.io/en/stable/installation/>`__.
+
+      Support for ``mongodb+srv://`` connection URIs requires dnspython.
+      For information on how to install dnspython, see the `dnspython
+      PyPi project page <https://pypi.org/project/dnspython/>`__.
+
+   .. step:: Create a Project Directory
+
+      In your shell, run the following command to create a 
+      directory called ``pymongo-quickstart`` for this project:
+
+      .. code-block:: bash
+
+         mkdir pymongo-quickstart
+
+      Run the following command to navigate into the project 
+      directory:
+
+      .. code-block:: bash
+
+         cd pymongo_quickstart
```

Comment:
Previous line of code uses a hyphen, not an underscore. not sure which is more idiomatic for python directories

```suggestion
         cd pymongo-quickstart
```